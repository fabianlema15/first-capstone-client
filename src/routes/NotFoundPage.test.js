import React from 'react';
import ReactDOM from 'react-dom';
import NotFoundPage from './NotFoundPage'
import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer';

describe('<NotFoundPage />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter>
      <NotFoundPage>
      </NotFoundPage>
    </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('Renders NotFoundPage as expected', () => {
	    const tree = renderer.create(<BrowserRouter>
        <NotFoundPage>
        </NotFoundPage>
      </BrowserRouter>).toJSON();
	    expect(tree).toMatchSnapshot();
	  });

})
