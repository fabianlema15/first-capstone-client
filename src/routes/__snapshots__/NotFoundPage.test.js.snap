// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> Renders NotFoundPage as expected 1`] = `
<section>
  <h2>
    404 - Page not found
  </h2>
  <p>
    Try going back to your previous page.
  </p>
</section>
`;
